controllers/datadogmonitor/controller.go

```diff
@@ -264,20 +264,31 @@ func (r *Reconciler) update(logger logr.Logger, datadogMonitor *datadoghqv1alpha
 	return nil
 }
 
-func (r *Reconciler) get(logger logr.Logger, datadogMonitor *datadoghqv1alpha1.DatadogMonitor, status *datadoghqv1alpha1.DatadogMonitorStatus, now metav1.Time) (datadogV1.Monitor, error) {
-	// Get monitor from Datadog and update resource status if needed
-	m, err := getMonitor(r.datadogAuth, r.datadogClient, datadogMonitor.Status.ID)
+func (r *Reconciler) get(logger logr.Logger, datadogMonitor *datadoghqv1alpha1.DatadogMonitor, now metav1.Time) (datadogV1.Monitor, error) {
+	// Get monitor from Datadog
+	return getMonitor(r.datadogAuth, r.datadogClient, datadogMonitor.Status.ID)
+}
+
+func updateMonitorStateAndDowntime(m datadogV1.Monitor, now metav1.Time, status *datadoghqv1alpha1.DatadogMonitorStatus, err error) {
 	if err != nil {
 		status.MonitorStateSyncStatus = datadoghqv1alpha1.MonitorStateSyncStatusGetError
-		return m, err
+		return
 	}
-	return m, nil
-}
 
-func updateMonitorState(m datadogV1.Monitor, now metav1.Time, status *datadoghqv1alpha1.DatadogMonitorStatus) {
 	convertStateToStatus(m, status, now)
 	status.MonitorStateLastUpdateTime = &now
 	status.MonitorStateSyncStatus = datadoghqv1alpha1.MonitorStateSyncStatusOK
+
+	if len(m.MatchingDowntimes) > 0 {
+		status.DowntimeStatus = datadoghqv1alpha1.DatadogMonitorDowntimeStatus{
+			IsDowntimed: true,
+			// Only show ID of first Downtime in the list
+			DowntimeID: int(m.MatchingDowntimes[0].Id),
```

Comment:
Why do we only want to show the first matching downtime ID? Is the first downtime ID the most recent downtime?

Reply:
we talked IRL, but it is arbitrary to just show the first one